tests/integration/test_spatial_methods/test_finite_volume.py

```diff
@@ -115,40 +114,6 @@ def get_error(n):
         rates = np.log2(err_norm[:-1] / err_norm[1:])
         np.testing.assert_array_less(1.99 * np.ones_like(rates), rates)
 
-    def test_cylindrical_div_convergence_quadratic(self):
```

Comment:
this test is not related to the main change in this PR, it should not be removed